iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/TreeFactory.java

```diff
@@ -272,6 +278,24 @@ public MultilineString multilineString(SyntaxToken openingTripleApostrophe, Synt
     return new MultilineStringImpl(openingTripleApostrophe, text, closingTripleApostrophe);
   }
 
+  public TypedLocalVariable typedLocalVariable(Identifier identifier, AmbientTypeReference primaryTypeExpression) {
+    return new TypedLocalVariableImpl(identifier, primaryTypeExpression);
+  }
+
+  public TypedVariableBlock typedVariableBlock(SyntaxToken lParen, Optional<SeparatedList<TypedLocalVariable, SyntaxToken>> variableList, SyntaxToken rParen) {
+    return new TypedVariableBlockImpl(lParen, variableList.or(emptySeparatedList()), rParen);
+  }
+
+  public TypedLambdaExpression typedLambdaExpression(TypedVariableBlock typedVariableBlock, AmbientTypeReference primaryTypeExpression, SyntaxToken doubleArrow,
+    Expression expression) {
```

Comment:
Nitpik, consider put all arguments in separate lines, for better readability (long line)